eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           npm install --prefix $HELIX_WORKITEM_PAYLOAD jsvu -g &&
           $HELIX_WORKITEM_PAYLOAD/bin/jsvu --os=linux64 --engines=v8 &&
           find ~/.jsvu -ls &&
-          ~/.jsvu/v8 -e "console.log(`V8 version: ${this.version()}`)"
+          (~/.jsvu/engines/v8/v8 -e "console.log(`V8 version: ${this.version()}`)" || echo)
```

Comment:
the echo is to suppress failure if v8 fails, right? but why?

Reply:
Yep. This change is only for `runtime-wasm-perf` optional pipeline, which is not working right now because `jsvu` fails to install the wrapper script, because we are using a very old node, and are unable to install a newer node on ubuntu 18.04 . So, I'm trying some workarounds to get the pipeline working to allow us to do quick test runs to check that the perf build won't break with a PR.